README.md

```diff
@@ -24,11 +24,19 @@ $ alto init --template static-website
 You can also apply the template to an existing Altostra project from Visual Studio Code by going to the Altostra view in the main toolbar and clickign on "Static Website" in the templates list.
 
 ## Deploying
-To deploy the project, run:
+To deploy the project, you must first create an image of it.
```

Comment:
Remove deploying completely from this readme. Let them use the docs and save us the split-brain on the docs.